tests/wpt/css-tests/selectors-3_dev/xhtml1/css3-selectors-lang-014.xht

```diff
@@ -27,7 +27,7 @@ This tests a detail related to :lang support. If :lang is not supported, a messa
 <script>
 test(function() {
 assert_equals(document.getElementById('colonlangcontroltest').offsetWidth, 0)
-assert_equals(document.getElementById('box').offsetWidth, 50);
+assert_equals(document.getElementById('box').offsetWidth, 100);
```

Comment:
So one thing to note: this test file is under selectors 3, where wildcard matching is not implemented. In a sense, it's great that we fail this test, because that means we actually follow selectors 4.
In other words, please revert this change and mark this test as failing. Same for the other ones that you've changed under selectors 3.